python/GafferSceneUI/LightEditor.py

```diff
@@ -612,7 +615,8 @@ def __buttonPress( self, pathListing, event ) :
 			menuDefinition.append(
 				"Edit...",
 				{
-					"command" : functools.partial( self.__editSelectedCells, pathListing, False )
+					"command" : functools.partial( self.__editSelectedCells, pathListing, False ),
+					"active" : pathListing.getSelection()[self.__soloColumnIndex].isEmpty(),
```

Comment:
Yes, I admit to burying my head in the sand on that one. I think of the "edit" button as "edit my underlying data directly" and since we're doing a pretty significant wrapper around that underlying data, I'm not entirely sure what the UI should be. I'm certainly open to discussing it.

Reply:
This is the most minimal change we can make to get a UI of some sort :

```Gaffer.Metadata.registerValue( Gaffer.Spreadsheet.RowPlug, "cells", "plugValueWidget:type", "GafferUI.LayoutPlugValueWidget" )```

But maybe this isn't the place to expose the user to the fact that the edit for this location is bundled up with edits for other locations too? Would we prefer to provide a simple checkbox that adds/removes the path behind the scenes? I suppose that could be done by registering a custom PlugValueWidget for the `cells` plugs on the SetMembershipProcessor?

Not sure how important it is that we pursue this immediately - would be good to get your thoughts @murraystevenson.

Reply:
For convenience trying it out, my latest push registers that metadata value and reactivates the "Edit" option in https://github.com/GafferHQ/gaffer/pull/5108/commits/f5920e52141cb4439ac027ff9a9a19fff869a68d

Reply:
After chatting about it offline, we decided to keep the "edit" item disabled because the UI it would expose is not intuitive.